indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -301,4 +267,111 @@
             }
         }).isEverythingSelected;
     };
+
+    function _fetchSourceCategory(categoryId) {
+        if (!_categories[categoryId]) {
+            _categories[categoryId] = categoryId;
```

Comment:
i guess you do that to avoid loading it twice? pretty ugly way to do that if you ask me ;)